documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -1,24 +1,25 @@
 Radix Sort
 ##########
 
----------------------------------------------------
-radix_sort and radix_sort_by_key Function Templates
----------------------------------------------------
+-----------------------------
+radix_sort Function Templates
+-----------------------------
 
-The ``radix_sort`` and ``radix_sort_by_key`` functions sort data using the radix sort algorithm.
-The sorting is stable, ensuring the preservation of the relative order of elements with equal keys.
-The functions implement a Onesweep* [#fnote1]_ algorithm variant. Both in-place and out-of-place
-overloads are provided. For out-of-place overloads, the input data order is preserved.
+The ``radix_sort`` function sorts data using the radix sort algorithm.
+The sorting is stable, preserving the relative order of elements with equal keys.
+There are in-place and out-of-place overloads. Out-of-place overloads do not alter the input sequence.
```

Comment:
I think I prefer the old 
"Both in-place and out-of-place overloads are provided."
to 
"There are in-place and out-of-place overloads."

However, its just my opinion, and is optional.